packages/invoices/src/utils.ts

```diff
@@ -102,7 +102,6 @@ export const isOutboundDisabled = (outbound: OutboundRowFragment): boolean => {
 export const isInboundDisabled = (inbound: InboundRowFragment): boolean => {
   return (
     inbound.status === InvoiceNodeStatus.Verified ||
-    inbound.status === InvoiceNodeStatus.Shipped ||
```

Comment:
This was wrong. Shipped is the status of the inbound when it is shown in the receiving store. If it's disabled here, you can't move to 'delivered'

Reply:
oh. oops, I missed that